build.yaml

```diff
@@ -4820,6 +4821,20 @@ targets:
   - grpc
   - gpr_test_util
   - gpr
+- name: vector_test
```

Comment:
you should also update the corresponding BUILD file

Reply:
It's already there.  See test/core/support/BUILD.